test/unit/data/model/migrations/test_migrations.py

```diff
@@ -1122,13 +1123,7 @@ def db_state6_gxy_state3_tsi_no_sam(url_factory, metadata_state6_gxy_state3_tsi_
 @pytest.fixture(autouse=True)
 def legacy_manage_db(monkeypatch):
     def get_alembic_cfg():
-        path = os.path.join(os.path.dirname(__file__), os.pardir, os.pardir, os.pardir, os.pardir, os.pardir)
```

Comment:
For something like this you can use something like... ``galaxy.util.resources.resource_path("galaxy.model.migrations", "alembic.ini")`` that should work uniformly across mega-git-Galaxy and standalone packages.